cfgov/ask_cfpb/views.py

```diff
@@ -93,12 +95,12 @@ def ask_search(request, language='en', as_json=False):
         results_page.result_query = ''
         return results_page.serve(request)
 
-    search = AskSearch(search_term=search_term, language=language)
-    if search.queryset.count() == 0:
+    search = AnswerPageSearch(search_term, language=language)
```

Comment:
We shouldn't make this change yet, as this is where we interact with ES 2.4

We should try and leave this code as is and implement an ES7 version that can be invoked within a flag check

Reply:
Done